packages/core/src/components/Banner/Banner.styles.tsx

```diff
@@ -1,24 +1,22 @@
-import styled from "@emotion/styled";
-import Snackbar, { snackbarClasses } from "@mui/material/Snackbar";
+import { CSSInterpolation } from "@emotion/serialize";
 import { theme } from "@hitachivantara/uikit-styles";
-import { transientOptions } from "@core/utils/transientOptions";
 
-export const StyledSnackbar = styled(
-  Snackbar,
-  transientOptions
-)(({ $isOpen }: { $isOpen: boolean }) => ({
-  [`&.${snackbarClasses.root}`]: {
-    ...($isOpen && {
+export const styles: {
+  root: CSSInterpolation;
+  anchorOriginTopCenter: CSSInterpolation;
+  anchorOriginBottomCenter: CSSInterpolation;
+} = {
```

Comment:
We should use something like `Partial<Record<keyof HvBannerClasses, CSSInterpolation>>` as proposed in the `ColorPicker` PR to avoid repeating the classe names. The same should be done for the remaining files.

Reply:
I was planning on opening a PR to apply this to the other components after the `ColorPicker` is merged.

Reply:
updated.

Reply:
You forgot to update the other files: message container, banner content, and action container :P